go.mod

```diff
@@ -30,6 +30,11 @@ require (
 	sigs.k8s.io/yaml v1.3.0
 )
 
+require (
+	github.com/Masterminds/sprig v2.22.0+incompatible // indirect
+	github.com/denisbrodbeck/machineid v1.0.1 // indirect
+)
+
```

Comment:
Why are these in a separated `require` section ?

Reply:
honestly I have no idea how this happened?